.travis.yml

```diff
@@ -15,6 +18,8 @@ matrix:
 
 before_script:
   - composer install --prefer-source
+  - npm install eslint eslint-config-wikimedia
+  - wget https://phar.phpunit.de/phpunit-4.8.phar
```

Comment:
I guess it would be good to pin PHPUnit version in composer.json. I am adding a commit to this PR.